HELP

```diff
@@ -32,3 +32,30 @@ Subcommands:
 Variables:
   IMG       The container image reference such as quay.io/freeipa/freeipa-operator:latest
             If IMG_TAG variable is not overrided, the local tag used is 'dev-GIT_SHA'.
+  SAMPLE    The configuration to be used using config/samples as base directory.
+            # make SAMPLE=ephimeral-storage recreate-idm-sample
```

Comment:
s/ephimeral/ephemeral/g (and in filenames too)

Reply:
Thanks! updated!